manager/controllers/utils/utils.go

```diff
@@ -118,6 +118,22 @@ func CreateDataSetIdentifier(datasetID string) string {
 	return id[:len(id)-1]
 }
 
+// Generating release name from step
+func GetReleaseName(applicationName string, namespace string, step app.FlowStep) string {
+	return GetReleaseNameByStepName(applicationName, namespace, step.Name)
+}
+
+// Generate release name from step name
+func GetReleaseNameByStepName(applicationName string, namespace string, stepName string) string {
+	fullName := applicationName + "-" + namespace + "-" + Hash(applicationName, 5) + "-" + Hash(namespace, 5) + "-" + stepName
```

Comment:
Why do we hash `applicationName` and `namespace` here if we already have the full value of them in the name?

Reply:
See above comment